src/core/evaluator.js

```diff
@@ -3519,7 +3521,6 @@ class EvalState {
   constructor() {
     this.ctm = new Float32Array(IDENTITY_MATRIX);
     this.font = null;
-    this.fontSize = 0;
```

Comment:
You definitely want to keep this line though, to ensure that there's always a meaningful default value.

(The changes in the `handleSetFont` method looks great, by the way.)